packages/ckbtc/src/minter.canister.ts

```diff
@@ -188,6 +190,26 @@ export class CkBTCMinterCanister extends Canister<CkBTCMinterService> {
       certified,
     }).retrieve_btc_status({ block_index: transactionId });
 
+  /**
+   * Returns the status of all BTC withdrawals for the user's main account.
+   *
+   * @param {bigint} transactionId The ID of the corresponding burn transaction.
+   * @param {boolean} certified query or update call
```

Comment:
Could you add the return type and a short explanation?

Reply:
done